java/ql/src/semmle/code/java/frameworks/Servlets.qll

```diff
@@ -337,3 +337,16 @@ predicate isRequestGetParamMethod(MethodAccess ma) {
   ma.getMethod() instanceof ServletRequestGetParameterMapMethod or
   ma.getMethod() instanceof HttpServletRequestGetQueryStringMethod
 }
+
+/**
+ * The method is `doGet` or `doPost`.
+ */
+class ServletRequestMethod extends Method {
```

Comment:
This nearly duplicates `isServletRequestMethod`; use that instead.

Reply:
@smowton Thanks for review, I made a change here.